pkg/statsd/handler_backend.go

```diff
@@ -0,0 +1,164 @@
+package statsd
+
+import (
+	"context"
+	"fmt"
+	"hash/adler32"
+	"sync"
+
+	"github.com/ash2k/stager/wait"
+	"github.com/sirupsen/logrus"
+
+	"github.com/atlassian/gostatsd"
+	"github.com/atlassian/gostatsd/pkg/statser"
+)
+
+// AggregatorFactory creates Aggregator objects.
+type AggregatorFactory interface {
+	// Create creates Aggregator objects.
+	Create() Aggregator
+}
+
+// AggregatorFactoryFunc type is an adapter to allow the use of ordinary functions as AggregatorFactory.
+type AggregatorFactoryFunc func() Aggregator
+
+// Create calls f().
+func (f AggregatorFactoryFunc) Create() Aggregator {
+	return f()
+}
+
+// BackendEventHandler dispatches metrics and events to all configured backends (via Aggregators)
+type BackendHandler struct {
+	eventWg          sync.WaitGroup
+	backends         []gostatsd.Backend
+	concurrentEvents chan struct{}
+
+	numWorkers int
+	workers    map[uint16]*worker
+}
+
+// NewBackendHandler initialises a new Handler which sends metrics and events to all backends
```

Comment:
All godoc strings must end with a dot. Try running the linter :)

Reply:
Dag nabbit.  I thought I caught this one.  I've already force pushed enough for today, so I'm ignoring this one :P